routers/repo/issue.go

```diff
@@ -673,7 +730,8 @@ func ViewIssue(ctx *context.Context) {
 	ctx.Data["RequireHighlightJS"] = true
 	ctx.Data["RequireDropzone"] = true
 	ctx.Data["RequireTribute"] = true
-	ctx.Data["RequireSimpleMDE"] = true
```

Comment:
rebase error (dont remove `ctx.Data["RequireSimpleMDE"] = true`) otherwhise we have a bug :(